client/src/components/pages/search-page/search-task/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import { FormInput, FormSelect, RankTag } from 'components';
+import styles from './search-task.module.scss';
+
+const sortByOptions = [
```

Comment:
Options configs can be moved to a separate file

Reply:
fixed